### PR TITLE
remove compiler warnings from ThreadUtils

### PR DESCRIPTION
**Pull request by PascalSchumacher**

### CODE REVIEW

File: src/main/java/org/apache/commons/lang3/ThreadUtils.java [hunk 1]
```diff
@@ -292,12 +292,12 @@ private AlwaysTruePredicate() {
         }
 
         @Override
-        public boolean test(@SuppressWarnings("unused") final ThreadGroup threadGroup) {
```
Comment (britter) [inline src/main/java/org/apache/commons/lang3/ThreadUtils.java]: Which IDE are you using? My IDE (IntelliJ 14.2) says this is okay.

Comment (PascalSchumacher) [inline src/main/java/org/apache/commons/lang3/ThreadUtils.java]: Eclipse 4.4.2 disagrees. ;)

Does IntelliJ require the suppression?

Comment (britter) [inline src/main/java/org/apache/commons/lang3/ThreadUtils.java]: No. But if I can make Eclipse users a litte bit more happy, I'll merge this for you :-)

**Comment by britter** [comment]
Thank you!

**Comment by PascalSchumacher** [comment]
Thanks for merging! :)

**Pull request merged**